### ISSUE TITLE
put example early on the page

### ISSUE THREAD

**Issue by davidelang** [rsyslog/rsyslog-doc#615]
up at the very top of the page there should be an example of a trivial, common use case, so that people don't have to page down to find the examples (and since they aren't always the last thing, they can't just go to the end of the page)

**Comment by deoren** [comment]
@davidelang Gotcha. That makes sense. I could see working in the most common, trivial use case directly at the end of each summary section as something that would be really useful.

**Comment by deoren** [comment]
> @davidelang: I could even see an argument for it being above the summary, especially if the summary is large.
> 
> the idea being that it should show up on the first screen of data, no matter
> what size screen you are using.

Understood. Re the summary though, it would probably be useful to have that be only a few sentences long (thinking of the module docs), then lead into the quick example. At some point on the page the longer description could be given where applicable (perhaps like the systemd section on the imuxsock doc).

**Comment by deoren** [comment]
Makes sense. This is good material for a docs "dev guide" if/when we get around to writing it (#451).
